### PR TITLE
[Enhancement] add mmaction.yml for test

### PR DESCRIPTION
**Pull request by irexyc**
## Motivation

add mmaction.yml for test

### CODE REVIEW

File: docs/zh_cn/04-supported-codebases/mmaction2.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ from mmdeploy.utils import get_input_shape, load_config
 import numpy as np
 import torch
 
-deploy_cfg = 'configs/mmaction/video-recognition/video-recognition_2d_onnxruntime_static.py'
+deploy_cfg = 'configs/mmaction/video-recognition/video-recognition_onnxruntime_static.py.py'
```
Comment (AllentDan) [inline docs/zh_cn/04-supported-codebases/mmaction2.md:127]: typo?

File: docs/en/04-supported-codebases/mmaction2.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ mim download mmaction2 --config tsn_r50_1x1x3_100e_kinetics400_rgb --dest .
 
 # convert mmaction2 model to onnxruntime model with dynamic shape
 python tools/deploy.py \
-    configs/mmaction/video-recognition/video-recognition_2d_onnxruntime_static.py \
+    configs/mmaction/video-recognition/video-recognition_onnxruntime_static.py.py \
```
Comment (RunningLeon) [inline docs/en/04-supported-codebases/mmaction2.md:82]: @irexyc still missing yml file for mmaction in https://github.com/irexyc/mmdeploy/tree/test-mmaction/tests/regression

**Review by AllentDan — COMMENTED** [review]
Hi, `mmaction.yaml` was not added into `tests/regression`?

File: tests/regression/mmaction.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      eval_name: top_k_accuracy
+      metric_key: top5_acc
+      tolerance: 1
+      multi_value: 100
+  convert_image: &convert_image
+    input_img: *video
+    test_img: *video
+  backend_test: &default_backend_test True
+  sdk:
+    sdk_dynamic: &sdk_dynamic ""
```
Comment (RunningLeon) [inline tests/regression/mmaction.yml:23]: pls remove `""`

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by AllentDan — APPROVED** [review]

**Pull request merged**